src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs

```diff
@@ -1272,7 +1272,8 @@ private void Selection_CollectionChanged(object sender, NotifyCollectionChangedE
 
         private void Controller_RequestsCrashPrompt(object sender, CrashPromptArgs args)
         {
-            var prompt = new CrashPrompt(args, dynamoViewModel);
+            var packageLoader = dynamoViewModel.Model.GetPackageManagerExtension().PackageLoader;
+            var prompt = new CrashPrompt(args, dynamoViewModel, packageLoader);
```

Comment:
How about adding the list of packages to `CrashPromptArgs`?

Reply:
Will check on that. That would also solve the API break right? Thanks for the comments

Reply:
> How about adding the list of packages to `CrashPromptArgs`?

That is worth looking at I agree